Add_Digits/add_digits.ts

```diff
@@ -1,3 +1,12 @@
 export function addUpDigits(num: number): number {
+    let numString:string = num.toString();
+    let arrNumbers = [];
+    let sum:number = 0;
+
+    for(let i:number = 0; i < numString.length; i++) {
+        arrNumbers.push(parseInt(numString.charAt(i)));
+        sum += arrNumbers[i];
+    }
+    return sum;
     throw new Error('Function is not implemented.');
```

Comment:
Ta bort alla dessa throw-statements, jag la bara in dem som ren boilerplate-kod, för att visa ett felmeddelande för de funktioner som ännu ej är implementerade. Jag kommer inte kommentera detta ytterligare, utan du får ta bort dem själv i de andra funktionerna.